### ISSUE TITLE
Model loss didn't reduce when training using official code

### ISSUE THREAD

**Issue by StevenXuf** [tanghme0w/AAAI26-HUG#2]
Hi authors,

We were trying to train the models using your code, so that it can be used as baselines. However, using the same training script with the same values for hyperparameters on the same split of the dataset, we inspected that the model did not learn too much from the dataset, as the loss almost did not drop (total loss from 21 for epoch 0 to 20 for epoch 30). Therefore, the trained final models were performing terribly, close to random guest. Can you check if your training script is right?

Plus, in the paper, it is not clear if the experimental results on CIRR were obtained using the validation or the test split.

We would be appreciated if you could clarify.
